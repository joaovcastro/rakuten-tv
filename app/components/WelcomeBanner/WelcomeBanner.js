import styled from 'styled-components';

const WelcomeBanner = styled.div`
  height: 500px;
  width: 100%;
  background: url('https://images-2.wuaki.tv/system/images/1072/original/visitor-generic-slider-web-1545911642-width1920-quality80.jpeg');
  background-position: center center;
  background-size: cover;
  background-repeat: no-repeat;
`;

export default WelcomeBanner;
