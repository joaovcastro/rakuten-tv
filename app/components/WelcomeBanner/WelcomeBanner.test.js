import React from 'react';
import { shallow } from 'enzyme';

import WelcomeBanner from './WelcomeBanner';

describe('<WelcomeBanner />', () => {
  it('correctly renders', () => {
    const wrapper = shallow(<WelcomeBanner />);
    expect(wrapper).toMatchSnapshot();
  });
});
