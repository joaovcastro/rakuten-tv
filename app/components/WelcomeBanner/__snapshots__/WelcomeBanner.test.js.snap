// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WelcomeBanner /> correctly renders 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  height: 500px;
  width: 100%;
  background: url('https://images-2.wuaki.tv/system/images/1072/original/visitor-generic-slider-web-1545911642-width1920-quality80.jpeg');
  background-position: center center;
  background-size: cover;
  background-repeat: no-repeat;
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
